src/site/stages/build/plugins/check-broken-links.js

```diff
@@ -52,15 +54,37 @@ function checkBrokenLinks() {
     });
 
     // Filter out drupal pages
-    const filteredFiles = Object.keys(files)
-      .filter(
-        fileName =>
-          !files[fileName].path || !files[fileName].path.includes('drupal'),
-      )
-      .reduce((acc, fileName) => {
-        acc[fileName] = files[fileName];
-        return acc;
-      }, {});
+    // Once Drupal is live, we should be able to delete all of this and just
+    // validate all of files.
+    let filteredFiles = { ...files };
+    if (DRUPALS.ENABLED_ENVIRONMENTS.has(buildOptions.buildtype)) {
+      if (DRUPALS.PREFIXED_ENVIRONMENTS.has(buildOptions.buildtype)) {
+        // On Drupal-prefixed builds, ignore all pages beginning with /drupal.
+        filteredFiles = Object.keys(files)
+          .filter(
+            fileName =>
+              !files[fileName].path || !files[fileName].path.includes('drupal'),
+          )
+          .reduce((acc, fileName) => {
+            acc[fileName] = files[fileName];
+            return acc;
+          }, {});
+      } else {
+        for (const fileName of filteredFiles) {
+          // On Drupal-enabled non-prefixed builds, either filter out the Drupal
+          // page; or, if this Drupal page overwrote a Vagov-content page, then
+          // validate the overwritten page.
+          const file = filteredFiles[fileName];
+          if (file.isDrupalPage) {
+            if (file.overwrittenVagovContentPage) {
+              filteredFiles[fileName] = file.overwrittenVagovContentPage;
+            } else {
+              delete filteredFiles[fileName];
+            }
+          }
+        }
+      }
+    }
```

Comment:
This is awful, but the issue is that Drupal-generated pages contain legitimate broken links on pages unique to Drupal, as well as on existing Healthcare pages. We need to fix this on the Drupal side, and patch this. Before the full live Drupal launch, we should be able to delete this whole filter.